src/components/HomePage.vue

```diff
@@ -84,41 +87,48 @@
     return listItems.value.length === 0;
   });
 
-  function searchToDo(searchString: string) {
-    if (searchString === '') {
-      displayItems.value = listItems.value;
-      isContentFiltered.value = false;
-      return;
+  onMounted(async () => {
+    try {
+      await todoStore.getToDos(sort.value, order.value);
```

Comment:
Please create a local function for getting the todos. The same code blocks is used in onMounted and searchTodo as well. The only one difference is the latest search param in the searchTodo().